packages/version/src/version-command.ts

```diff
@@ -603,26 +615,78 @@ export class VersionCommand extends Command<VersionCommandOption> {
       })
     );
 
-    chain = chain.then(() =>
-      // update modern lockfile (version 2 or higher) when exist in the project root
-      loadPackageLockFileWhenExists(rootPath)
-        .then(lockFileResponse => {
-          if (lockFileResponse && lockFileResponse.lockfileVersion >= 2) {
-            for (const pkg of this.packagesToVersion) {
-              this.logger.silly(`lock`, `updating root "package-lock-json" for package "${pkg.name}"`);
-              updateTempModernLockfileVersion(pkg, lockFileResponse.json);
+    // update the project root lock file, we will read and write back to the lock file
+    // this is currently the default update and if none of the flag are enabled (or all undefined) then we'll consider this as enabled
+    if (this.options.manuallyUpdateRootLockfile || (this.options.manuallyUpdateRootLockfile === undefined && !this.options.packageLockfileOnly)) {
```

Comment:
This is only executed for npm, right? Where is the check?

Reply:
fixed